parsons/bluelink/__init__.py

```diff
@@ -0,0 +1,7 @@
+from parsons.bluelink.bluelink import Bluelink
+from parsons.bluelink.person import Person, Email, Address, Phone, Identifier, Tag, Score
+
+__all__ = [
+    'Bluelink',
+    'Person', 'Email', 'Address', 'Phone', 'Identifier', 'Tag', 'Score'
```

Comment:
I'm not sure if it's a good idea to put these here, or if they are here if they should have names like BluelinkPerson instead of Person. Will spend a bit of time looking at this.

Reply:
Okay, I think the way I have it here is good.

Parsons has an (I think optional) convention where you can export out your package's classes from parsons' own package.
see: https://github.com/BluelinkData/parsons/blob/master/parsons/__init__.py

 It would allow you to do this:
`from parsons import Bluelink, Person ...`

instead of:
`from parsons.bluelink import Bluelink, Person`

Doing that loses some of the benefits of namespacing. A name like Person is quite likely to clash with another class in the future. So then all our classes would need to be renamed (BluelinkPerson, BluelinkEmail....).

I don't think we need to export our classes, so I'm going to stick with these names.